x-pack/plugins/rule_registry/server/alert_data_client/tests/update.test.ts

```diff
@@ -181,33 +187,26 @@ describe('update()', () => {
         outcome: 'unknown',
         type: ['change'],
       },
-      message: 'User is updating alert [id=1]',
+      message: 'User is updating alert [id=NoxgpHkBqbdrfX07MqXV]',
```

Comment:
You have a test case for an `update()` that results in a successful audit event; what about an audit error when the user is unauthorized?

Reply:
Yup very true. Added an audit failure test here: 36838ab2e10ba9b2640665345758373075e0fa44